### PR TITLE
Update 'custom animation' examples for bpmn-visualization@0.11.0

### PR DESCRIPTION
**Pull request by aibcmars**
**Title**
Update demos under the custom animation section to use the new API available in 0.11.0

https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/chore-update_demos_custom_animation/examples/index.html

### CODE REVIEW

**Review by tbouffard — APPROVED** [review]
Tested on Ubuntu 18.04 with Chromium 87.0.4280.141 and Firefox 84.0.2

File: examples/custom-animation/growing-sequence-flow/README.md [hunk 1]
```diff
@@ -33,9 +32,5 @@ It looks like `<path d="..." fill="none" stroke="black" stroke-width="1.78" stro
 ````
 
 ```javascript
-    const animatedSequenceFlowElt = bpmnVisualization.bpmnElementsRegistry.getElementsByIds(['sequence_flow_id'])[0].htmlElement;
-    animatedSequenceFlowElt.classList.add('growing');
+    bpmnVisualization.bpmnElementsRegistry.addCssClasses(['flow_5'], 'growing');
```
Comment (csouchet) [inline examples/custom-animation/growing-sequence-flow/README.md:35]: For the readme, I prefer to keep as id  **'sequence_flow_id'** for the sequence flow. 
The user of the lib doesn't know what is and where is the id 'flow_5'.

File: examples/custom-animation/running-dashed-message-flow/README.md [hunk 2]
```diff
@@ -32,9 +32,5 @@ It looks like `<path d="..." fill="none" stroke="black" stroke-width="1.78" stro
 ````
 
 ```javascript
-    const animatedMessageFlowElt = bpmnVisualization.bpmnElementsRegistry.getElementsByIds(['message_flow_id'])[0].htmlElement;
-    animatedMessageFlowElt.classList.add('running-dashed');
+    bpmnVisualization.bpmnElementsRegistry.addCssClasses(['message_4'], 'running-dashed');
```
Comment (tbouffard) [inline examples/custom-animation/running-dashed-message-flow/README.md:35]: ❓ I guess we have the same remark here about the element id

Comment (csouchet) [inline examples/custom-animation/running-dashed-message-flow/README.md:35]: Same as previous comment

Comment (aibcmars) [reply examples/custom-animation/running-dashed-message-flow/README.md]: ah yes I get it :)

**Review by csouchet — APPROVED** [review]

**Pull request merged**